pages/_app.tsx

```diff
@@ -21,6 +21,8 @@ function MyApp({ Component, pageProps }: AppProps) {
     <Provider>
       <Head>
         <title>writedown - markdown notes app</title>
+        <link rel="manifest" href="/manifest.json" />
+        <link rel="apple-touch-icon" href="/logo.png" />
```

Comment:
We should add these (along with 16px and 32px favicons:
```suggestion
        <link rel="apple-touch-icon" href="/logo.png" />
        <meta name="theme-color" content="#317EFB" />
        <link
          href="/icons/favicon-16x16.png"
          rel="icon"
          type="image/png"
          sizes="16x16"
        />
        <link
          href="/icons/favicon-32x32.png"
          rel="icon"
          type="image/png"
          sizes="32x32"
        />
```